ipi/inputs/forcefields.py

```diff
@@ -831,3 +833,225 @@ def fetch(self):
             active_thresh=self.active_thresh.fetch(),
             active_out=self.active_output.fetch(),
         )
+
+
+class InputFFCavPhSocket(InputForceField):
```

Comment:
wouldn't be better to use InputFFSocket as a parent class to avoid code repetition?
see below